tmt/utils.py

```diff
@@ -3410,6 +3463,15 @@ def __init__(
 # A type representing compatible sources of keys and values.
 KeySource = Union[Dict[str, Any], fmf.Tree]
 
+NormalizeCallback = Callable[[Any], T]
+
+
+def normalize_string_list(value: Union[None, str, List[str]]) -> List[str]:
```

Comment:
Please, add a docstring.

Reply:
Fixed in e5f266bef1df4b011ed20040575f778189aa001f